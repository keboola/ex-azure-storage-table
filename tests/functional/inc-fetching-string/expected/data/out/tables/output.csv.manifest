{
    "primary_key": [
        "id"
    ],
    "columns": [
        "id",
        "string"
    ],
    "incremental": true
}
