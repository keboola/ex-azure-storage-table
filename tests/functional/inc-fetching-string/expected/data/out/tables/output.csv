"1","00001"
"20","00020"
"300","00300"
"4000","04000"
"50000","50000"
